### ISSUE TITLE
Local plugins not working correctly

### ISSUE THREAD

**Issue by danieldspx** [nrwl/nx#12541]
## Current Behavior
My local plugin can not be loaded. It results in 
```
Plugin listed in `nx.json` not found: ./tools/plugins/my-plugin/index.js

Cannot find module './tools/plugins/my-plugin/index.js'
```

## Expected Behavior
It should load the local plugin as expected and described in the [Extending the Project Graph of Nx](https://nx.dev/recipe/project-graph-plugins#extending-the-project-graph-of-nx) documentation.

## Steps to Reproduce
Make sure you have at least one project defined in your workspace.json and just follow the guide and you will not be able to make it work:
[Extending the Project Graph of Nx](https://nx.dev/recipe/project-graph-plugins#extending-the-project-graph-of-nx)

### Environment
Ubuntu
Node v16.17.1
Nx 14.8.4 <-- This is the problematic guy

**Comment by danieldspx** [comment]
Closing because https://github.com/nrwl/nx/pull/12406 seems to fix the issue.

**Comment by danieldspx** [comment]
I am reopening this issue because although @AgentEnder fixed this 8 days ago, the 14.8.4 that happened 12 hours ago does not include this fix. I think that it should be great to publish a new version again picking this fix @FrozenPandaz.

**Comment by AgentEnder** [comment]
We are working on getting v15 out ASAP, which will be off master and contain this fix. The issues get closed as the PRs are merged, not on release, so I'm going to close this one out again.
